miqi/runtime/protocol_specs.py

```
@@ -372,6 +373,15 @@ def spec(
     result_model=SESSION_METHOD_RESULT_MODELS["sessions.rename"],
 )
 
+# ── files (Phase 30 handlers, #790) ────────────────────────────────────────
+
+FILES_CHECK_MANY = model_spec(
+    "files.check_many",
+    FilesCheckManyParams,
+    scope=MethodScope.SESSION,
+    description="资产面板路径可达性批量校验（渲染资产卡片时按需调用）",
```

Ruff (0.16.2)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)